pkl-doc/src/test/files/DocGeneratorTest/output/com.package1/1.2.3/baseModule/BaseClass.html

```diff
@@ -39,8 +39,8 @@ <h1 id="declaration-title">BaseClass<span id="declaration-version">1.2.3</span><
         <h2 class="member-group-title">Methods<span class="toggle-inherited-members">(<span class="toggle-inherited-members-link button-link">show inherited</span>)</span></h2>
         <ul>
           <li>
-            <div id="getClass()" class="anchor"> </div>
-            <div class="member inherited expandable hidden collapsed"><a class="member-selflink material-icons" href="#getClass()">link</a>
+            <div id="getClass%28%29" class="anchor"> </div>
```

Comment:
`id`s are not URIs. Except when the attribute is itself a URI/URL should it be escaped as such. In other cases, this should be XML escaped.

Reply:
Good catch!

Reply:
Actually, I just tested this, and the browser doesn't treat XML escapes the same way.

`https://example.com/foo#%3E` won't navigate to an anchor with ID `"&gt;"`, but it _will_ navigate to an anchor of ID `"%3E".